src/System.Management.Automation/engine/hostifaces/ConnectionFactory.cs

```diff
@@ -67,72 +65,12 @@ public static Runspace CreateRunspace(PSHost host)
                 throw PSTraceSource.NewArgumentNullException("host");
             }
 
-            return CreateRunspace(host, RunspaceConfiguration.Create());
+            return new LocalRunspace(host, InitialSessionState.CreateDefault2());
```

Comment:
Same here, if we want to mimic the existing behavior, then we should use `CreateDefault`.

Reply:
will change